spec/spec.ddoc

```diff
@@ -3,6 +3,36 @@ ROOT = ..
 SEARCHDEFAULT_SPEC = selected
 DDOC_BLANKLINE = $(DIVC blankline)
 
+BODY_PREFIX =
```

Comment:
I'm surprised. This is a straightforward application - the macro is there for being overridden, and that's what this does. Where is the hack?